### DISCUSSION TITLE
Flood inundation mapping with mRM

### DISCUSSION THREAD

**Discussion by cuneyd** [mhm-ufz/mHM#84] [Q&A] [answered]
Dear mHM and mRM experts,

I think the mRM flux file provides us the discharge in channels and mHM flux file provides runoff maps. I wonder if mHM allow users to create inundation maps at L1/L11 scale like in other hydrodynamic models (hec-ras, lisflood).

**Comment by rohinikumar82** [comment, marked as answer]
No - mHM/mRM provides flow information and not water levels which you would be required to create inundation maps.  For the latter, you would need detailed stream geometry ...
